traffic_portal/README.md

```diff
@@ -21,4 +21,5 @@
 
 An AngularJS client served from a lightweight Node.js web server. Traffic Portal was designed to consume the Traffic Ops API.
 
-Installation / configuration instructions are found at https://github.com/apache/trafficcontrol/tree/master/traffic_portal/build
+Installation / configuration instructions are found at:
+* https://github.com/apache/trafficcontrol/tree/master/traffic_portal/build
```

Comment:
This renders as a list (as above), but also GitHub's markdown processor provides certain extensions, one of which could be useful here. Specifically, when linking to a repository-internal location you can use relative paths. Here's what I would suggest:

```markdown
Installation / configuration instructions may be found in [the `build/` directory](./build/)
```

At any rate, I'd discourage the use of naked URLs, at least when they're that long.